modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerPartitionReconciliationExtendedTest.java

```diff
@@ -216,6 +258,31 @@ public void testWorkWithAllSetOfCachesIfParameterAbsent() throws Exception {
         assertEquals(usedCaches.size(), setOfCaches.size());
     }
 
+    /**
+     * Creates internal and custom caches and runs reconciliation cmd with args.
+     */
+    private Set<String> getUsedCachesForArgs(String... args) throws Exception {
+        Set<String> usedCaches = new HashSet<>();
+        LogListener lsnr = fillCacheNames(usedCaches);
+        log.registerListener(lsnr);
```

Comment:
I think you want to unregister the log listener at the end of this method

Reply:
Fixed